site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -352,6 +354,21 @@ export const WorkspaceReadyPage: FC<WorkspaceReadyPageProps> = ({
 				}
 			/>
 
+			{/* Cancel confirmation dialog */}
```

Comment:
I don't think we need this comment. The code below is pretty easy to understand.